resources/idlharness.js

```diff
@@ -1333,7 +1333,8 @@ function IdlInterface(obj, is_callback, is_mixin)
 
     /** An array of IdlInterfaceMembers. */
     this.members = obj.members.map(function(m){return new IdlInterfaceMember(m); });
-    if (this.has_extended_attribute("Unforgeable")) {
+    if (this.has_extended_attribute("Unforgeable") ||
```

Comment:
We should probably file a follow-up to remove this again in the future, right?

Reply:
My plan was to use https://github.com/web-platform-tests/wpt/issues/23148 to track that; do you think it would be better as a separate issue?

Related question; would you prefer if there was a TODO in the code linked to that same issue? (I was concerned about sprinkling ~dozen identical TODOs everywhere 'LegacyXYZ' has now been added).

Reply:
No that's fine. And I'm okay with not tracking it as a TODO but please include the link in the commit message so you can tell easily from blame.